cs.go

```diff
@@ -0,0 +1,195 @@
+
+package cmd
+
+import (
+	"fmt"
+	"net"
+	"os"
+	"os/signal"
+	"strconv"
+	"time"
+
+	"github.com/spf13/cobra"
+)
+
+// ncsCmd represents the ncs command
```

Comment:
```suggestion
// csCmd represents the cs command
```
This is the `cs` command.